pages/osx/launchctl.md

```diff
@@ -23,10 +23,10 @@
 
 `launchctl unload ~/Library/LaunchAgents/{{my_script}}.plist`
 
-- Manually run a known (loaded) agent/daemon, even if it isn’t the right time (note: this command uses the agent's label, rather than the filename):
+- Manually run a known (loaded) agent/daemon, even if it is not the right time (note: this command uses the agent's label, rather than the filename):
 
 `launchctl start {{my_script}}`
 
-- Manually kill the process associated with a known agent/daemon, if it's running:
```

Comment:
I don't think this was necessary. We have a lot of places with "it's" in our repo.

Reply:
I noticed that, and I agree it's not necessary, but I let it go in because it is equivalent, arguably more "explicit", and the other changes were worthwhile. Sorry if I jumped the gun :sweat_smile: I have a sweet spot for such cleanup changes, as a [wikignome](https://en.wikipedia.org/wiki/Wikipedia:WikiGnome) myself.

Reply:
Haha, it's just that this change stood out to me :laughing: . I think we should maintain a balance between usefulness and verbosity. It is not a huge deal, but in future PRs, let us go with whatever the author chooses to use.